features/make-phar.feature

```diff
@@ -29,3 +29,30 @@ Feature: Check `utils/make-phar.php` output
     Then the return code should be 1
     And STDOUT should be empty
     And STDERR should be empty
+
+  Scenario: Phar renaming affects template path resolution
+    Given an empty directory
+    And a new Phar with the same version
+    And a WP installation
+    And I run `wp plugin install generic-example-plugin --activate`
```

Comment:
```suggestion
    And I run `wp plugin install https://github.com/wp-cli-test/generic-example-plugin/releases/download/v0.1.1/generic-example-plugin.0.1.1.zip --activate`
```

Reply:
Btw, I replaced the usual `duplicate-post` because it now requires WP 6.6+, and that won't work for some of our tests. Given that we're also adding caching to our GitHub downloads, this will do.